Sources/MongoSwift/AsyncAwait/MongoClient+AsyncAwait.swift

```diff
@@ -48,8 +48,8 @@ extension MongoClient {
      * Run the `listDatabases` command.
      *
      * - Parameters:
-     *   - filter: Optional `Document` specifying a filter that the listed databases must pass. This filter can be based
-     *     on the "name", "sizeOnDisk", "empty", or "shards" fields of the output.
+     *   - filter: Optional `BSONDocument` specifying a filter that the listed databases must pass. This filter can be
```

Comment:
I ended up doing some general cleanup of docstrings throughout the new API methods during this. I noticed some remaining instances of `Document` in existing API methods as well which these were copied from but I am going to make a separate PR to fix those up.